app/src/androidTest/java/com/github/sdp_begreen/begreen/fragments/ProfileDetailsFragmentTest.kt

```diff
@@ -521,7 +521,7 @@ class ProfileDetailsFragmentTest {
             displayName = "User Test 1",
             email = "user1@email.ch",
             phone = "1984z719848",
-            profilePictureMetadata = PhotoMetadata("VaRgQioAuiGtfDlv5uNuosNsACCJ_profile_picture")
+            profilePictureMetadata = ProfilePhotoMetadata("VaRgQioAuiGtfDlv5uNuosNsACCJ_profile_picture", null, null)
```

Comment:
We can ommit the two `null`s here

Reply:
Made the change

Reply:
Sorry for the confusion. I made the change in line 66 for the following line of code. I confused the two...

`private val userProfilePicturePhotoMetadata = ProfilePhotoMetadata("user1_profile_picture")`

Now it's fixed for both

Reply:
Sorry for the confusion. I made the change in line 66 and forgot this one. Now it's fixed for both